Source/JavaScriptCore/runtime/JSGlobalObject.h

```diff
@@ -1060,6 +1062,10 @@ class JSGlobalObject : public JSSegmentedVariableObject {
     {
         m_requiresTrustedTypes = required;
     }
+    void setTrustedScriptStructure(Structure* structure)
+    {
+        m_trustedScriptStructure.set(vm(), this, structure);
+    }
```

Comment:
If we do configuring this in global object constructor, we do not need this function.